src/Microsoft.CSharp/src/Microsoft.CSharp.csproj

```diff
@@ -9,9 +9,11 @@
     <ExcludeLocalizationImport Condition="'$(TargetsWindows)' != 'true'">true</ExcludeLocalizationImport>
   </PropertyGroup>
   <!-- Default configurations to help VS understand the configurations -->
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
+  <PropertyGroup Condition = "'$(Configuration)|$(Platform)' == 'netstandard_Debug|AnyCPU'" >
+    <BuildConfiguration>netstandard</BuildConfiguration>
```

Comment:
You shouldn't need to set BuildConfiguration in these blocks. When building an individual project the Configuration and Platform should be enough. We might need to do some initialization of BuildConfiguration in a common location so the bin-placing works as expected.